validphys2/src/validphys/filters.py

```diff
@@ -488,7 +516,7 @@ def get_cuts_for_dataset(commondata, rules) -> list:
 
     Parameters
     ----------
-    commondata: NNPDF CommonData spec
+    commondata: :py:class:`validphys.coredata.CommonData`
```

Comment:
Sphinx is inferring the `:py:class:` part here, if you wish you can simplify with
```suggestion
    commondata: validphys.coredata.CommonData
```